dbt_subprojects/dex/models/trades/apechain/dex_apechain_base_trades.sql

```diff
@@ -0,0 +1,49 @@
+{{ config(
+    schema = 'dex_apechain'
+    , alias = 'base_trades'
+    , materialized = 'view'
+    )
+}}
+
+{% set base_models = [
+    ref('saru_apechain_base_trades')
+    , ref('camelot_apechain_base_trades')
+] %}
+
+WITH base_union AS (
+    SELECT *
+    FROM (
+        {% for base_model in base_models %}
+        SELECT
+            blockchain
+            , project
+            , version
+            , block_month
+            , block_date
+            , block_time
+            , block_number
+            , token_bought_amount_raw
+            , token_sold_amount_raw
+            , token_bought_address
+            , token_sold_address
+            , taker
+            , maker
+            , project_contract_address
+            , tx_hash
+            , evt_index
+        FROM
+            {{ base_model }}
+        {% if not loop.last %}
+        UNION ALL
+        {% endif %}
+        {% endfor %}
+    )
+)
+
+{{
+    add_tx_columns(
+        model_cte = 'base_union'
+        , blockchain = 'apechain'
+        , columns = ['from', 'to', 'index']
+    )
+}}
```

Comment:
Since apechain is not initiated yet, this macro will fail for now.

Reply:
you actually don't need `from` and `to` here anymore, they are on the decoded table. Unfortunately `index` is not

Reply:
this is pretty engrained in dex pipeline for now. i'd rather do one effort to clean this up, rather than here in a specific chain use case